crates/toml/src/table.rs

```diff
@@ -8,7 +8,7 @@ use crate::Value;
 
 /// Type representing a TOML table, payload of the `Value::Table` variant.
 /// By default it is backed by a BTreeMap, enable the `preserve_order` feature
-/// to use a LinkedHashMap instead.
+/// to use an IndexMap instead.
```

Comment:
Instead of specifying the data structures used, could you instead update it to specify the behavior (sorted by key, respect insertion-order)

Reply:
Should I also remove the reference to `BTreeMap` then and describe both just in terms of ordering (for consistency)?

Reply:
Yes, that'd be great!